### DISCUSSION TITLE
Is there possibility to remake windoes with AGS?

### DISCUSSION THREAD

**Discussion by clsty** [end-4/dots-hyprland#292] [General]
I'm not requesting for this, just wondering how difficult it might be to remake "windoes" with AGS, since the appearances are totally different between i-i and windoes. And if it's difficult, I'd like to know what's the problem that is hard to solve.

Thanks in advanced.

**Comment by end-4** [comment]
it's going to be much much easier and less hacky than using eww
but currently i want to focus on the quality of illogical impulse first

**Comment by midn8hustlr** [comment]
I guess backend we can keep same as i-i, only frontend and styling we have to change. There were some extra modules in windoes, like the game bar/ screenshot bar, audio mixer, performance widget and the dock. All of these widgets we can add to i-i also at sometime. The overview widgets we need to add menu, and that will be pretty much, ig.

Not sure if its possible, or how difficult will it be, but can we segregate the styling or frontend aspect in ags, so that we can have multiple frontends for one backend? This will help to create multiple themes, like windows or mac while keeping the core functionality intact.
